extractor/filesystem/os/dpkg/dpkg_test.go

```diff
@@ -139,6 +139,49 @@ func TestFileRequired(t *testing.T) {
 			wantRequired:     true,
 			wantResultMetric: stats.FileRequiredResultOK,
 		},
+		{
+			name:             "status file",
+			path:             "usr/lib/opkg/status",
+			wantRequired:     true,
+			wantResultMetric: stats.FileRequiredResultOK,
+		},
+		{
+			name:         "status as a directory",
+			path:         "usr/lib/opkg/status/foo",
+			wantRequired: false,
+		},
+		{
+			name:             "status file required if file size < max file size",
```

Comment:
The file sizes are already tested above, no need to include test cases for them again